src/app/api/[[...route]]/route.ts

```diff
@@ -27,6 +27,26 @@ app.get('/hello', (c) => {
   });
 });
 
+app.get('/whoami', async (c) => {
+  const session = await auth();
```

Comment:
I believe it's good to have the `/api/whoami` route before deploying to production because it allows us to debug the session easily. So, I added a comment to warn this route that this should be deleted before the production.

Reply:
By the way, I thought the `/api/hello` was no longer needed, so I deleted it.